liblouis/compileTranslationTable.c

```diff
@@ -3514,8 +3523,12 @@ compileRule(FileInfo *nested, TranslationTableOffset *newRuleOffset,
 								return 0;
 							}
 						}
-					if (!addRule(nested, opcode, &ruleChars, &ruleDots, after, before,
-								newRuleOffset, newRule, noback, nofor, table))
+					TranslationTableRule *r;
+					if (addRule(nested, opcode, &ruleChars, &ruleDots, after, before,
```

Comment:
I don't quite understand why you are doing something in the if when the call to `addRule` failed.

Reply:
You mean the `ok = 0`?